docs/s3.md

```diff
@@ -0,0 +1,76 @@
+# Performance Test Result storage in S3 bucket for SLV
+
+**NOTE: This document is only relevant to puppet employees**
+
+The SLV team has an s3 bucket for storing results from performance testing.
+
+`
+s3://slv-performance-results
+`
+
+## Convention
+
+S3 is [object storage](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html),
+so it doesn't have true directories.  But it does like to make it look like it does.  The convention 
+is to use names like `foo/bar`  and `foo/car` to group files together.  They are really two separate
+files that exist next to each other, but with the cli you can `ls foo/` and you will get two
```

Comment:
Sorry... now the extra "two" needs to be removed: "...and you will get two both" should be "...and you will get both" :)

Reply:
lols, caffeine was slow to kick in. :)  Fix now.